src/styles/themes/reset.scss

```diff
@@ -90,6 +90,7 @@
   display: block;
   height: 1.2em;
   width: auto;
+  margin-left: 0.3rem;
```

Comment:
technically this is adding it as well for the non-text version, but they use the same dom so it's very hard to know. A little extra margin may not be that big of a deal?

Reply:
Could we add a `margin-right` to `.ais-PoweredBy-text` instead? Since it's only present in RIS?

Reply:
yes, that's probably better :)

Reply:
Thanks! Should we keep the `margin-left` on the Algolia theme? Not sure why it's here in the first place

Reply:
`margin-right` on `.ais-PoweredBy-text` for both themes is probably the way, yes.

One potential edge case I see is if someone uses one of the themes and is overriding the `margin-left` on `.ais-PoweredBy-logo` with their value in a user rule. Now have a wider gap.

I guess this is quite unlikely, and it's difficult to draw the line of breaking vs. non-breaking here anyway.